presto-native-execution/scripts/dockerfiles/README.md

```diff
@@ -48,8 +48,9 @@ podman compose build centos-native-dependency
 ### 3. Build Runtime Image
 
 Specify the build type using ``BUILD_TYPE`` (defaults to Release)
-environment variable.
+build argument.
```

Comment:
What are the options for ``BUILD_TYPE``, and are they documented anywhere? I find ``Release`` mentioned here, and elsewhere ``Debug``. Are there others?

Reply:
I add an explanation for this.